test/elements.js

```diff
@@ -49,7 +49,7 @@ test('svg', function (t) {
   </svg>`
   t.equal(result.tagName, 'svg', 'create svg tag')
   t.equal(result.childNodes[0].tagName, 'rect', 'created child rect tag')
-  t.equal(result.childNodes[1].getAttribute('xlink:href'), '#test', 'created child use tag with xlink:href')
+  t.equal(result.childNodes[2].getAttribute('xlink:href'), '#test', 'created child use tag with xlink:href')
```

Comment:
That's the tricky case that I tried to address this in the last paragraph of the PR.

The trouble is differentiating, say:

```js
bel`
  <svg width="150" height="100" viewBox="0 0 3 2">
    <rect width="1" height="2" x="0" fill="#008d46" />
    <use xlink:href="#test" />
  </svg>
`
```

from:

```js
bel`
  <p>
    <strong>Goal</strong>
    <em>Do the right thing.</em>
  </p>
`
```

As a practical matter, any HTML element can end up with `white-space: preserve`, and need space as it was written.  This PR tries to address the most common case, `<pre>`.  Unless we start using other tag names to intelligently guess when space between elements should be removed, we have to make a trade-off that affects both of my examples above.